frontend/src/Pages/Signup.jsx

```diff
@@ -6,7 +6,7 @@ import { faEye, faEyeSlash } from "@fortawesome/free-solid-svg-icons";
 import validate from "../../common/validation.js";
 import axios from "axios";
 import { toast } from "react-toastify";
-import useGoogleAuth from "../../common/useGoogleAuth"
+
 
```

Comment:
Same comment here as well.
All code related to Google signing is removed can you please fix it

Reply:
Sure!!